flink-connectors/flink-connector-kafka/src/main/java/org/apache/flink/streaming/connectors/kafka/table/SinkBufferFlushMode.java

```diff
@@ -18,10 +18,11 @@
 
 package org.apache.flink.streaming.connectors.kafka.table;
 
+import java.io.Serializable;
 import java.util.Objects;
 
 /** Sink buffer flush configuration. */
-public class SinkBufferFlushMode {
+public class SinkBufferFlushMode implements Serializable {
```

Comment:
🙈

Reply:
I don't get it why we need to make it Serializable here?

Reply:
Objects of this class are passed to the `ReducingUpsertSink` which is instantiated by the jobmanager and needs to be serialized before it is sent to the taskmanagers to create the writer instances.

In general, classes that are passed to the sink should be serializable.